packages/react-slider/src/utils/getMarkPercent.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Gets the current percentage position for the marks with respect to adjacent marks.
+ *
+ * The marks percentage position relative to their individual positions.
+ * @param markValues
```

Comment:
Seems like this might have been an unfinished combination of multiple comments? Also it would be good to add some info about the intended use of the return value.
```suggestion
 * Gets the current percentage position for the marks with respect to adjacent marks.
```